_posts/2021-03-15-debezium-1-5-beta2-released.adoc

```diff
@@ -0,0 +1,150 @@
+|`STRING` +
+The unique identifier of the signal such as a UUID.
+|TYPE
+|`STRING` +
+The unique command to be performed.
+|DATA
+|`STRING` +
+The payload for the command.
+|===
+
+Lastly, the signal table must be explicitly found as part of your connector's include/exclude-list specifications.
+In other words, if you're specifying a list of tables to monitor, this list will need to be adjusted to include the name of the signal table as well.
+
+This new feature has quite a number of use cases that we intend to explore in future releases.
+Lets discuss what signals are currently supported in this release.
+
+==== Log Signal
+
+The log signal is detected when the `TYPE` column in the signal table is `log`.
+This signal requests that Debezium write the contents of the `DATA` column (payload) to the connector logs as-is.
+This can be useful for a variety of purposes from debugging to tracking progress of database script operations and much more.
+
+As an example:
+
+```sql
+INSERT INTO DEBEZIUM_SIGNALS (ID, TYPE, DATA) VALUES ('1', 'log', 'Hello World');
+```
+
+Once that insert is committed and written to the database's transaction logs,
+Debezium will recognize the signal and will then write `Hello World` to the connector logs using the `INFO` log level.
+
+==== Schema Changes Signal
+
+The schema changes signal is detected when the `TYPE` column in the signal table is `schema-changes`.
+This signal tells Debezium to emit a `SchemaChangeEvent` to the schema changes topic that is based on the changes supplied in the row's `DATA` column (payload).
+The format of the `DATA` column must be given in JSON and an example of the format is below.
+Additionally, this signal will also have Debezium update it's in-memory representation of the table's schema structure.
+
+As an example, we have a PostgreSQL table `s1.a` where we want to add a new column `aa`:
+
+```json
+{
+  "database": "postgres",
+  "changes": [{
+    "type": "ALTER",
+    "id": "s1.a",
+    "table": {
+      "defaultCharsetName": null,
+      "primaryKeyColumnNames": [ "pk" ],
+      "columns": [{
+        "name": "pk",
+        "jdbcType": 4,
+        "nativeType": 23,
+        "typeName": "serial",
+        "typeExpression": "serial",
+        "charsetName": null,
+        "length": 10,
+        "scale", 0,
+        "position": 1,
+        "optional": false,
+        "autoIncremented": true,
+        "generated": false
+      }, {
+        "name": "aa",
+        "jdbcType": 4,
+        "nativeType": 23,
+        "typeName": "int4",
+        "typeExpression": "int4",
+        "charsetName": null,
+        "length": 10,
+        "scale", 0,
+        "position": 2,
+        "optional": true,
+        "autoIncremented": false,
+        "generated": false
+      }]
+    }
+  }]
+}
+```
+
+With this JSON payload, the signal would be inserted as:
+
+```sql
+INSERT INTO DEBEZIUM_SIGNALS (ID, TYPE, DATA) VALUES ('1', 'schema-changes', <json-payload-string>);
+```
+
+== Other Features and Fixes
```

Comment:
Nothing else of interest to mention, out of 56 issues?

Reply:
I added a few more things that seemed noteworthy.  If there is something specific you'd like me to add let me know as most of the changes were either documentation, bugfixes, or general polish afaict.